src/DistributedLockCodeGen/DistributedLockCodeGen.csproj

```diff
@@ -10,9 +10,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="nunit" Version="3.12.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+    <PackageReference Include="nunit" />
+    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
```

Comment:
Note that this used to have a different version than `DistributedLock.Tests.csproj`. There is no `VersionOverride` per [this thread](https://github.com/madelson/DistributedLock/pull/186#discussion_r1485634980), but that means that there are extra diffs in the lock file.